examples/standalone/lrauv_control/lrauv_control.py

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright (C) 2023 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+#
+# Check README for detailed instructions.
+# Usage:
+#   $ gz sim -r worlds/lrauv_control_demo.sdf
+#   $ # ./lrauv_control speed_m_s yaw_rad pitch_rad
+#   $ python3 ./lrauv_control.py 0.5 0.78 0.174
+#
+
+from gz.msgs10.double_pb2 import Double
+from gz.msgs10.odometry_pb2 import Odometry
+from gz.math7 import Quaterniond, Vector3d
+from gz.transport13 import Node
+
+from threading import Lock
+
+import math
+import sys
+import time
+
+
+# Helper class for the PID controller for
+# linear speed, pitch and yaw angles.
+class Controller:
+    def __init__(self):
+        # Mutex to synchronize internal variables.
+        self.controllerMutex = Lock()
+
+        # Desired state.
+        self.targetSpeed = 0
+        self.targetYawAngle = 0
+        self.targetPitchAngle = 0
+
+        # Errors
+        self.errorSpeed = 0
+        self.errorSpeedIntegral = 0
+        self.errorYawAngle = 0
+        self.errorPitchAngle = 0
+
+        # States to be tracked and controlled.
+        self.speed = 0
+        self.yawAngle = 0
+        self.pitchAngle = 0
+
+        # PID gains and error limits.
+        # PI for speed.
+        self.kSpeed = -30
+        self.kISpeed = -0.5
+        self.errorSpeedIntegralMax = 10
+
+        # P for yaw and pitch control.
+        self.kYawAngle = -0.5
+        self.kPitchAngle = 0.6
+
+    # Set the target states to be tracked,
+    # i.e. linear speed (m/s), pitch and yaw angles (rad).
+    def SetTargets(self, _speed, _yaw, _pitch):
+        with self.controllerMutex:
+            if _speed == 0 and (_yaw != 0 or _pitch != 0):
+                print("Speed needs to be non zero for non zero")
```

Comment:
The `print` message should say `"Speed needs to be non zero for non zero pitch and yaw angles"`

Reply:
Thanks! Not sure what happened here since it doesn't even make sense what was written before haha. Done in https://github.com/gazebosim/gz-sim/pull/2044/commits/80ff6f73b17428a77a26a68a414a145ea7f793c4.